misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/LocalDateFeatureAssertionsSpec.kt

```diff
@@ -33,6 +34,8 @@ abstract class LocalDateFeatureAssertionsSpec(
     month: Fun1<LocalDate, Expect<Int>.() -> Unit>,
     dayOfWeekFeature: Feature0<LocalDate, DayOfWeek>,
     dayOfWeek: Fun1<LocalDate, Expect<DayOfWeek>.() -> Unit>,
+    dayFeature: Feature0<LocalDate, Int>,
```

Comment:
A detail but if you don't mind, change the order here as well, in other places you have year, month, day, dayOfWeek